gcn_classic_to_json/notices/SWIFT_BAT_GRB_POS_ACK/__init__.py

```diff
@@ -1,10 +1,79 @@
+import numpy as np
+
 from ... import utils
 
 
 def parse(bin):
+    bin[15]  # Unused. According to docs: '4 bytes for the future'
+    bin[19]  # Unused. Flags are either internal or equivalent to bin[18]
+    bin[26:36]  # Unused. According to docs: '40 bytes for the future'
+    bin[36]  # Unused. Flags Equivalent to bin[18]
+    bin[38]  # Unused. Sun/Moon parameters
+
+    integ_time = bin[14] * 16 / 1000
+
+    lat, lon = bin[16:17].view(">i2")
+
+    soln_status_bits = np.flip(np.unpackbits(bin[18:19].view(dtype="u1")))
+
+    flag_descriptions = {
+        0: "A point source was found.\n",
+        1: "It is a GRB.\n",
+        2: "It is an interesting src.\n",
+        3: "It is in the flight catalog.\n",
+        5: "It is definitely not a GRB.\n",
+        6: "It is probably not a GRB or Transient(hi bkg level).\n",
+        7: "It is probably not a GRB or Transient(low image significance; < 7).\n",
+        8: "It is in the ground catalog.\n",
+        9: "It is probably not a GRB or Transient(negative bkg slop).\n",
+        10: "StraTracker not locked so trigger porbably bogus.\n",
+        11: "It is probably not a GRB or Transient(very low image significance; < 6.5).",
+        12: "It is the catalog of sources to be blocked.\n",
+        13: "There is a bright star nearby.\n",
+        14: "This was orginally a SubTresh, but it is now converted to a real BAT_POS.\n",
+        15: "This is a source that has purposefully been removed from on-board catalog.\n",
+        16: "This matched a Nearby_Galaxy in the on-board catalog.\n",
+        28: "There was a temporal coincidence with another event.\n",
+        29: "There was a spatial coincidence with another event.\n",
+        30: "This is a test submission",
+    }
+    comments = "".join(
```

Comment:
```suggestion
    comments = "\n".join(
```

and remove the `\n` characters from the dictionary above.